### ISSUE TITLE
Feature request: Automatic notifications and alerts of pools for delegates

### ISSUE THREAD

**Issue by rodrigonzalez** [papacarp/pooltool.io#71]
In the user area, a delegate could receive various notifications, such as
- Block created.
- Slot loaded.
- Epoch summary.
- Maintenance of the pool or change in its status.
- Changes to the pool fees.
- Stake changes.
- Battles.
- Sync problems.
... etc.

If and only if the delegate decides to do so. To do so, you could access a set of options where you can choose which alerts to receive and which not to receive.

**Comment by acewf** [comment]
wouldn't you get most of this from Pooltool bot? https://t.me/pooltoolbot

**Comment by rodrigonzalez** [comment]
Yes, but the central place from which to set up alarms and warnings should be the website. Then the user could choose to receive notifications by website, telegram, mail, sms, etc.
